api/namex/services/name_request/name_request.py

```diff
@@ -217,8 +215,9 @@ def map_request_data(self, name_request, map_draft_attrs=False):
         # If this is a draft, set name request header attributes
         if map_draft_attrs:
             name_request = self.map_draft_attrs(name_request)
-            name_request = self.map_request_header_attrs(name_request)
-            name_request = self.map_request_comments(name_request)
+
+        name_request = self.map_request_header_attrs(name_request)
+        name_request = self.map_request_comments(name_request)
 
         try:
```

Comment:
This try block doesnt do anything?

Reply:
It does:
```
        try:
            if new_state_code == State.COND_RESERVE:
                name_request.consentFlag = 'Y'

            if new_state_code in [State.RESERVED, State.COND_RESERVE]:
                name_request.expirationDate = self.create_expiry_date(
                    start=name_request.submittedDate,
                    expires_in_days=56,
                    tz=timezone('UTC')
                )
        except Exception as err:
            raise MapRequestDataError(err)
```

Reply:
ok.  use the create_expiry_date to set the priority date.